src/main/kotlin/minesweeper/entity/MineMap.kt

```diff
@@ -0,0 +1,14 @@
+package minesweeper.entity
+
+class MineMap {
+  fun createMap(mapInfo: MapInformation): List<MapElement> {
+    return mutableListOf<MapElement>().apply {
+      repeat(mapInfo.numberOfMines) { this.add(MapElement.MINE) }
```

Comment:
아래와 같이  repeat없이도 작성이 가능합니다!

```suggestion
      addAll(List(mapInfo.numberOfMines) { MapElement.MINE })
```

Reply:
오! 이런 방법이 있군요. 감사합니다.